andriy.safronov/02.rb

```diff
@@ -0,0 +1,4 @@
+a=0
+hash = {'Yes' => 23, 'b' => 'travel', 'yesterday' => 34, 5 => '234', yesss: :fg, try: 30, key: 'some value', ' yesterday1 '=> 34,' yesteryear '=> 2014}
+hash.each {|key, value| a=a+1 if key.to_s.downcase.start_with? "yes" }
```

Comment:
instead of introducing a new variable for calculation, you can use #count  method with block.
```suggestion
matched_keys_count  = hash.count { |key, value| key.to_s.downcase.start_with? "yes" }
```